File: app/src/main/java/com/rtukpe/fixtures/data/model/LeagueTable.java
package com.rtukpe.fixtures.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class LeagueTable {

    @SerializedName("leagueCaption")
    public String league;

    @SerializedName("matchday")
    public int matchday;

    @SerializedName("standing")
    public ArrayList<Standing> standing;

    public class Standing {

        @SerializedName("rank")
        public int rank;

        @SerializedName("team")
        public String team;

        @SerializedName("teamId")
        public int id;

        @SerializedName("playedGames")
        public int playedGames;

        @SerializedName("crestURI")
        public String logo;

        @SerializedName("points")
        public int points;

        @SerializedName("goals")
        public int goals;

        @SerializedName("goalsAgainst")
        public int goalsAgainst;

        @SerializedName("goalDifference")
        public int goalDifference;
    }
}

File: app/src/main/java/com/rtukpe/fixtures/data/model/LeagueTable.kt
package com.rtukpe.fixtures.data.model

import com.google.gson.annotations.SerializedName
import java.util.*

class LeagueTable {

    @SerializedName("leagueCaption")
    lateinit var league: String

    @SerializedName("matchday")
    @JvmField
    var matchday: Int = 0

    @SerializedName("standing")
    lateinit var standing: ArrayList<Standing>

    inner class Standing {

        @SerializedName("rank")
        @JvmField
        var rank: Int = 0

        @SerializedName("team")
        var team: String? = null

        @SerializedName("teamId")
        @JvmField
        var id: Int = 0

        @SerializedName("playedGames")
        @JvmField
        var playedGames: Int = 0

        @SerializedName("crestURI")
        var logo: String? = ""

        @SerializedName("points")
        @JvmField
        var points: Int = 0

        @SerializedName("goals")
        @JvmField
        var goals: Int = 0

        @SerializedName("goalsAgainst")
        @JvmField
        var goalsAgainst: Int = 0

        @SerializedName("goalDifference")
        @JvmField
        var goalDifference: Int = 0
    }
}
